CHANGES/889.feature

```diff
@@ -0,0 +1 @@
+Added `--header` parameter to allow passing a custom header along with every request.
```

Comment:
```suggestion
Added `--header` parameter to allow passing an arbitrary number of custom headers along with every request.
```

Reply:
You can even override the user agent string that way now.